src/App.js

```diff
@@ -1,26 +1,13 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
```

Comment:
Remove these two files `App.css` and `logo.svg`, since they are not needed and they shouldn't leak on master. Same goes for the following files:
`public/favicon.ico`
`public/logo512.png`